client/client.go

```diff
@@ -80,6 +82,7 @@ func newClient(rootURL *url.URL, timeout time.Duration, isInsecure bool) Client
 				TLSClientConfig: &tls.Config{InsecureSkipVerify: isInsecure},
 			},
 		},
+		jwt: config.GetJWT(),
```

Comment:
The idea was to not use config package in this one. As you can see it brings unnecessary complexity into tests and has this hidden env variable which needs to be documented.

Would be much better to have a constructor which accepts jwt token and creates appropriate client

Reply:
Add a new constructor and you need to change the code of every pack that wants to use JWT. Keep as is and all you need to do is provide an extra env var or not.

You're right, it does add a bit of complexity to the tests, but not that much in the grand scheme of things? I'm happy to change this if people want me to